### PR TITLE
Merge/foundation release/1.13.1

### PR DESCRIPTION
**Pull request by meowsbits**

### CODE REVIEW

**Comment by meowsbits** [comment]
Latest test failure is a timeout (20m) - https://github.com/etclabscore/core-geth/actions/runs/6240148619/job/16939591748?pr=563#step:4:1202

File: build/ci.go [hunk 1]
```diff
@@ -153,6 +153,13 @@ var (
 	// we need to switch over to a recursive builder to jumpt across supported
 	// versions.
 	gobootVersion = "1.19.6"
+
+	// This is the version of execution-spec-tests that we are using.
+	// When updating, you must also update build/checksums.txt.
+	executionSpecTestsVersion = "1.0.2"
+
+	// This is where the tests should be unpacked.
+	executionSpecTestsDir = "tests/spec-tests"
```
Comment (ziogaschr) [inline build/ci.go:162]: Interesting addition. @meowsbits, based on your recent checks, do we have anything cooked already in spec-tests?

Comment (meowsbits) [reply build/ci.go:162]: No. But its interesting that the test runner is `execBlockTest` -- https://github.com/etclabscore/core-geth/blob/merge/foundation-release/1.13.1/tests/block_test.go#L75

**Review by ziogaschr — APPROVED** [review]
LGTM. Nice work @meowsbits

**Comment by ghost** [comment]
@meowsbits Any ETA to merge this branch?

**Comment by meowsbits** [comment]
Review done, closing in favor of https://github.com/etclabscore/core-geth/pull/593.

**Pull request closed without merging**